recipe/meta.yaml

```diff
@@ -19,8 +19,7 @@ requirements:
     - make  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    # 20210526 csz temporarily disable OpenMP on MacOS
-    #    - llvm-openmp  # [osx]
+    - llvm-openmp  # [osx]
```

Comment:
@czender, I'm putting back llvm-openmp for OSX.  Let me know if we should still skip it.